### DISCUSSION TITLE
General Settings: "Use Project Color in App" Option

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3122] [Feature Requests]
We decided not to use the project color (eg: brand color) for the App's primary color, since many brand colors simply won't work. For instance, A company with RED branding would end up with all their normal buttons as RED... making them look dangerous. However, there is a good chance that your brand color **would** work well within the App, and it would be nice to offer that level of customization in the App (without CSS).

I propose either:
1. A toggle field to use the Project Color as the primary CSS variable color
2. A separate color field to choose the primary color from within the App
    * This allows the brand to be RED, but the primary to be a separate, complimentary brand color

Thoughts?

**Comment by rijkvanzanten** [comment]
I think I prefer Nº2: having a dedicated _primary color_ option. I can very well imagine that brands have a logo color, and a dedicated primary color for use in their tools and assets. You wouldn't want the logo color to be used as primary in those cases

**Reply by benhaynes** [reply]
OK, that sounds good!

Maybe we extend this a bit, to allow for overriding the MAIN App colors? (eg: theming, but in the App). We'd just have to choose which colors/variables we want to dynamically support.

**Reply by rijkvanzanten** [reply]
I like the sound of that. Just gotta work out how / if that's going to work in dark mode

**Reply by benhaynes** [reply]
True, though most of the applications I've seen disable dark mode when you apply a theme.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
